config/archetype/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>org.kuali.rice</groupId>
     <artifactId>rice-config</artifactId>
-    <version>ua-2.5-release1</version>
+    <version>2.5.19-uar11-SNAPSHOT</version>
```

Comment:
Jeff, uar is typically reserved for UAccess Research, short of KC for University of Arizona.
Can we eplace uar with just ua? Also, as far as I'm aware, we haven't had release1 for ua-rice yet so why 11?